Mathlib/GroupTheory/Congruence/Basic.lean

```diff
@@ -69,6 +69,61 @@ protected def congr {c d : Con M} (h : c = d) : c.Quotient ≃* d.Quotient :=
   { Quotient.congr (Equiv.refl M) <| by apply Con.ext_iff.mp h with
     map_mul' := fun x y => by rcases x with ⟨⟩; rcases y with ⟨⟩; rfl }
 
+@[simp]
+theorem congr_mk {c d : Con M} (h : c = d) (a : M) :
+    Con.congr h (a : c.Quotient) = (a : d.Quotient) := rfl
+
+@[to_additive]
+theorem comap_conGen_of_Bijective {M N : Type*} [Mul M] [Mul N] (f : M → N)
+    (hf : Function.Bijective f) (H : ∀ (x y : M), f (x * y) = f x * f y) (rel : N → N → Prop) :
```

Comment:
~~I am surprised you need bijectivity here. I would expect this be true when `f` is not ...~~ Nevermind I am being silly